--- openais-0.82/Makefile~	2007-09-09 09:00:13.000000000 +0200
+++ openais-0.82/Makefile	2007-12-18 19:55:18.268577936 +0100
@@ -138,7 +138,7 @@
 	install -m 644 exec/service.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
 	install -m 644 exec/timer.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
 	install -m 644 exec/objdb.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
-	install -m 644 exec/print.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
+	install exec/logsys.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
 	install -m 644 exec/config.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
 	install -m 644 include/swab.h $(DESTDIR)$(INCLUDEDIR_SERVICE)
 	install -m 644 man/*.3 $(DESTDIR)$(MANDIR)/man3
